README.md

```diff
@@ -64,3 +65,62 @@ loaders = Dict({
     val: { rn: loader, rf: loader, sn: loader, sf: loader}
 })
 ```
+
+### losses
+
+`trainer.losses` is a dictionary mapping to loss functions to optimize for the 3 main parts of the architecture: generator `G`, discriminators `D`, domain classifier `C`:
+
+```python
+trainer.losses = {
```

Comment:
@adrienju so you know, this needs to be updated as we work on losses

Reply:
Oh right ! I forgot to add doc here thanks !